src/ocamlorg_frontend/layouts/package_layout.eml

```diff
@@ -224,6 +224,7 @@ Layout.base
           a.appendChild(comment);
 
           a.addEventListener("mousedown", (event) => goToUrl(event, href));
+          a.addEventListener("touchstart", (event) => goToUrl(event, href));
```

Comment:
touchstart should work on mobile devices. I tested it using an emulator since I don't have access to an apple device, and it worked as expected.